configure.ac

```diff
@@ -13,6 +13,11 @@ AC_DEFINE(VMA_LIBRARY_MINOR, vma_ver_minor, [VMA Minor Version])
 AC_DEFINE(VMA_LIBRARY_REVISION, vma_ver_revision, [VMA Revision])
 AC_DEFINE(VMA_LIBRARY_RELEASE, vma_ver_release, [VMA Release])
 
+GIT_VER=`git describe --long --abbrev=40 --dirty --tags 2> /dev/null || echo ""`
+if test -n "$GIT_VER"; then GIT_VER=`echo $GIT_VER | sed -e 's/-dirty/+/' | sed s/.*-g//`; else  GIT_VER=""; fi
+
+AC_DEFINE_UNQUOTED(VMA_GIT_VERSION, "${GIT_VER}", [VMA Git Version])
+
```

Comment:
Why `git rev-parse --short=7 HEAD` does not work for you? It provides the same result I think.
Consider simple solution:
`AC_DEFINE_UNQUOTED(VMA_GIT_VERSION, "$(git rev-parse --short=7 HEAD)", [VMA Git Version])`